diesel/src/macros/associations/belongs_to.rs

```diff
@@ -45,15 +45,25 @@
 ///
 /// [custom_derive]: https://crates.io/crates/custom_derive
 ///
-/// ```ignore
+/// ```no_run
+/// # #[macro_use] extern crate diesel;
+/// # #[macro_use] extern crate custom_derive;
+/// # table! { users { id -> Integer, } }
+/// # table! { posts { id -> Integer, user_id -> Integer, } }
+/// # custom_derive!{
+/// #     #[derive(Identifiable(users))]
+/// #     pub struct User {
+/// #         id: i32,
+/// #     }
+/// # }
 /// custom_derive! {
-///     #[derive(BelongsTo(User, foreign_key = user_id)]
-///     #[table_name(posts)]
+///     #[derive(BelongsTo(User, foreign_key = user_id, table = posts), Identifiable(posts))]
```

Comment:
The file defining  the custom_derive [macro](https://github.com/DanielKeep/rust-custom-derive/blob/master/src/lib.rs) didn't change in the last 5 month according to the github history.
Also testing the code provided as documentation results in a compiler error. So this definitively needs to change.
Do you have any better ideas?